### PR TITLE
55 51 56 gui and label fixes

### PR DESCRIPTION
**Pull request by InMyOrbit**
Solved issues #51, #55 & #56. 

Plus added input check on temporary storage label text.

### CODE REVIEW

**Comment by emanuelen5** [comment]
Looks good, just want to test run

**Comment by emanuelen5** [comment]
I test ran and found/fixed a bug (that appeared after previous merge with master). I however have two observations when creating labels for firebox and temporary storage (**FYI, I created the labels 2020-03-15**): 

## Label for firebox
image

### Change suggestions
1. I think it should say
    > any member can use this product ***from*** &lt;date&gt;

    to make it clearer when the end date is.
2.   There should be a title and/or icon on the label so that it cannot be used as/confused with a temporary storage label.

## Label for temporary storage
image

### Question
The date is set to only today + 30 days: Shouldn't this be 90 days?

File: src/label/creator.py [hunk 1]
```diff
@@ -218,7 +229,7 @@ def create_chemical_storage_label(member_id, name):
     instruction_text = f'Any member can use this product after'
```
Comment (emanuelen5) [inline src/label/creator.py]:
```suggestion
    instruction_text = f'Any member can use this product from'
```

File: src/label/creator.py [hunk 2]
```diff
@@ -204,7 +215,7 @@ def create_box_label(member_number, name):
     return label_image
 
 
-def create_chemical_storage_label(member_id, name):
+def create_fire_box_storage_label(member_id, name):
```
Comment (emanuelen5) [inline src/label/creator.py]: Add a logo/title like "Fire box item", as for [temporary storage labels](https://github.com/makerspace/memberbooth/blob/16ba30c2b02fb4d8fd551c9f7267e9cabaf91eb8/src/label/creator.py#L65-L66)

File: src/label/creator.py [hunk 3]
```diff
@@ -26,7 +26,7 @@
 JSON_VERSION_KEY = 'v'
 
 TEMP_STORAGE_LENGTH = 30
```
Comment (emanuelen5) [inline src/label/creator.py]: Isn't temporary storage allowed to be lying around for 90 days?

```suggestion
TEMP_STORAGE_LENGTH = 90
```

Comment (InMyOrbit) [reply src/label/creator.py]: We discussed this and agreed to lower this limit, but I think to 45.

File: src/label/creator.py (hunk 2 above)
Comment (InMyOrbit) [reply src/label/creator.py]: I agree.

File: src/label/creator.py (hunk 3 above)
Comment (InMyOrbit) [reply src/label/creator.py]: image

Fixed label according to instruction.

**Pull request merged**